control_toolbox/include/control_toolbox/pid_ros.hpp

```diff
@@ -312,7 +382,7 @@ class PidROS
    *               If not stated explicitly using "/" or "~", prefix is interpreted as global, i.e.,
    *               "/" will be added in front of topic prefix
    */
-  void set_prefixes(const std::string & topic_prefix);
+  [[deprecated]] void set_prefixes(const std::string & topic_prefix);
```

Comment:
We are removing this setter? Is this used anywhere in our codebase? I would be surprised if it is.